Code_analyzer.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+#include "Code_parser.h"
+#include "File_finder.h"
+
+class Code_analyzer
+{
+	private:
+		int total_lines
+			, total_code_lines
```

Comment:
Please create a new variable with type name and initialize it here.

Reply:
Resolved